apps/analytics/src/modules/__init__.py

```
@@ -4,3 +4,4 @@
 from .aggregated_course_analytics import *
 from .participant_performance import *
 from .instance_activity_performance import *
+from .activity_progress import *
```

Ruff (0.8.0)

`from .activity_progress import *` used; unable to detect undefined names

(F403)